### PR TITLE
Build on container agents

### PR DESCRIPTION
**Pull request by NotMyFault**
Container agents are usually faster to provision and generate less cost for the project.

### CODE REVIEW

**Comment by jetersen** [comment]
Don't think compilation for windows is needed as it is a API library

**Comment by NotMyFault** [comment]
> Don't think compilation for windows is needed as it is a API library

That complies with the current configuration, happy to remove it if you want.

**Comment by jetersen** [comment]
That's only because the default is both. I honestly thought it was only Linux 😅 As in the new default is.

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env groovy
 
 /* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin(jdkVersions: [11])
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 11],
+    [platform: 'windows', jdk: 11],
```
Comment (jetersen) [inline Jenkinsfile:8]:
```suggestion
```

**Pull request merged**